app/components/App/App.js

```diff
@@ -18,7 +17,9 @@ class App extends Component {
             <Route exact path='/' component={Play} />
             <Route exact path='/instructions' component={Instructions}/>
             <Route path='/learn' component={Learn} />
-            <p className='copyright-notice'>&#169; 2017 Leta Keane</p>
+            <footer>
+              <p className='copyright-notice'>&#169; 2017 Leta Keane</p>            
+            </footer>
```

Comment:
@Jeff-Duke The ```footer``` was added just in my eval this morning :) I'd like it to be on every page, so I'm not sure if it's worth making a component, especially since it's just that one line? But ... I can always use more practice writing components!

Also, I will TOTALLY change App to a const; it was just a class in the boilerplate I used and I never got around to editing it. Thank you!